<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetUserSettings</class>
 <widget class="QWidget" name="WidgetUserSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  color: white;
  /*background-color: rgb(50,82,95);*/
}
QGroupBox {
  border-radius: 0px;
  border: 3px solid rgb(131, 147, 167);
  background-color: transparent; /*rgb(80,112,125);*/
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QTextEdit {
  background-color: rgb(101,117, 137);  
}
QPlainTextEdit {
  background-color: rgb(101,117, 137);  
}
QComboBox {
  background-color: rgb(101,117, 137);  
}
QCheckBox::indicator {
  border: 2px solid rgb(80,112,125);
}
QCheckBox::indicator:checked {
  background-color: white;
}
QListView {
  background-color: rgb(101,117, 137);  
}
QTableWidget {
  background-color: rgb(101,117, 137);  
}
QDateTimeEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background: rgb(101,117, 137);
}
QPushButton::checked {
  border: 2px solid white;
  font-weight: bold;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditName">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>32</number>
                </property>
                <property name="placeholderText">
                 <string>Your name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditLogin">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>32</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditEmail">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>32</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonPhoto">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>128</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>128</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">#pushButtonPhoto {
  background-color: transparent;
}
#pushButtonPhoto:hover {
  background-color: transparent;
}</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/application.qrc">
                <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>124</width>
                <height>124</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Change Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Current Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPwCurrent">
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>32</number>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Enter your current password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPw">
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>32</number>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Enter your password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Repeat Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPwRepeat">
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>32</number>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Enter your password</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onBtnPhotoClicked()</slot>
 </slots>
</ui>
